### PR TITLE
Add testplan for course deletion

### PR DESCRIPTION
**Pull request by maexled**
Part of Gamify-IT/issues#214

### CODE REVIEW

**Review by WuestFlorian — APPROVED** [review]

File: dev-manuals/test-plans/lecturer-interface/u-lecturer-interface-16-delete-course.md
Comment (delvh) [inline dev-manuals/test-plans/lecturer-interface/u-lecturer-interface-16-delete-course.md:14]:
```suggestion
At least one course is created. The tester is on the view of the course (`/courses/<course-id>`).

## Postcondition
```

**Review by delvh — APPROVED** [review]

**Review by cdingm — APPROVED** [review]

**Comment by delvh** [comment]
Oh wait, I probably should have blocked until the other PR is merged...
Oh well, as long as no one tests until the other PR has been merged, everything should be fine.

**Pull request merged**
